client/package.json

```diff
@@ -26,8 +26,8 @@
         "ios:copy:release": "npm run native:build && npx cap copy ios",
         "ios:open": "npm run ios:copy && npx cap open ios",
         "electron:copy": "npm run native:build && npx cap copy @capacitor-community/electron",
-        "electron:copy:dev": "npm run native:build -- --mode test && npx cap copy @capacitor-community/electron",
-        "electron:open": "npm run electron:copy && npx cap open @capacitor-community/electron",
+        "electron:copy:dev": "dotenv -c development -- npm run native:build -- --mode test && npx cap copy @capacitor-community/electron",
+        "electron:open": "npm run electron:copy:dev && npx cap open @capacitor-community/electron",
         "electron:open:dev": "npm run electron:copy:dev && npx cap open @capacitor-community/electron",
```

Comment:
What the difference between `electron:open` and `electron:open:dev` ? :raised_eyebrow: